sources/packages/web/src/components/layouts/aest/AESTHomeSideBar.vue

```diff
@@ -40,6 +40,18 @@
         </v-list-item-content>
       </v-list-item>
     </v-list>
+    <template v-slot:append>
```

Comment:
I believe that we have been using the shortcut lately than the extended version `<template #append>`. Can we stick to it?